### DISCUSSION TITLE
How can I show a notification after a redirection?

### DISCUSSION THREAD

**Discussion by scajal** [filamentphp/filament#9532] [Help]
I'm redirecting from an internal class in my app to a Filament resource's page, and I want to show a notification.

**Comment by leandrocfe** [comment]
Maybe using the mount method to send the notification?

**Reply by binaryfire** [reply]
@scajal Did you find a simple solution for this? I'm redirecting from a controller to a panel. I've tried `return redirect('/admin')->with('success', 'My success message.');` but it doesn't work.

**Reply by scajal** [reply]
@binaryfire I just showed the notification without redirecting the user.

**Reply by leandrocfe** [reply]
@scajal @binaryfire,

1 - Create a custom dashboard page
https://filamentphp.com/docs/3.x/panels/dashboard#customizing-the-dashboard-page

2 - Create a mount method in the new dashboard page

```php

namespace App\Filament\Pages;

use Filament\Notifications\Notification;

class Dashboard extends \Filament\Pages\Dashboard
{
    public function mount(): void
    {
        if (request()->session()->has('success')) {
            Notification::make()
                ->title(request()->session()->get('success'))
                ->send();
        }
    }
}

```

This should show the notification if you redirect to the /admin

```php
return redirect('/admin')->with('success', 'My success message');
```

**Comment by ghost** [comment]
I am doing this from `App\Exceptions\Handler` and it does not seem to be working... any ideas?
